app/src/main/java/com/nervousfish/nervousfish/modules/pairing/APairingHandler.java

```diff
@@ -37,15 +38,15 @@
      * {@inheritDoc}
      */
     @Override
+    @DesignedForExtension
```

Comment:
So that FindBugs knows that this method is designed for extension and won't complain because it isn't final